src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__ap_assessment_dashboard.sql

```diff
@@ -27,38 +27,39 @@ with
     )
 
 select
-    e.student_number,
-    e.lastfirst,
-    e.enroll_status,
-    e.cohort,
     e.academic_year,
+    e.academic_year_display,
     e.region,
     e.schoolid,
-    e.school_abbreviation,
+    e.school,
+    e.student_number,
+    e.student_name,
     e.grade_level,
-    e.advisor_lastfirst,
-    e.is_enrolled_recent,
-    e.is_enrolled_y1,
+    e.enroll_status,
     e.entrydate,
     e.exitdate,
+    e.cohort,
+    e.advisory,
+    e.is_enrolled_recent,
+    e.is_enrolled_y1,
     e.is_504 as c_504_status,
     e.lep_status,
+    e.gifted_and_talented,
+    e.contact_id,
+    e.ktc_cohort,
 
     s.cc_dateenrolled as ap_date_enrolled,
     s.cc_dateleft as ap_date_left,
     s.sections_external_expression as ap_course_section,
     s.teacher_lastfirst as ap_teacher_name,
     s.ap_course_subject,
 
-    adb.id as contact_id,
-    adb.kipp_hs_class as ktc_cohort,
-
     a.administration_round,
     a.test_date,
     a.scale_score,
     a.rn_highest,
 
-    if(e.spedlep in ('No IEP', null), 0, 1) as sped,
+    if(e.iep_status = 'No IEP', 0, 1) as sped,
```

Comment:
maybe "is_sped_int" for clarity's sake?

Reply:
I dont want to break this view just yet. I need to redo it anyway because I am changing the data source from salesforce to illum later.